### ISSUE TITLE
Switch to runtime-only build?

### ISSUE THREAD

**Issue by ThaNarie** [Blocklevel/blue#77]
https://github.com/Blocklevel/blue/blob/0e775075a9e0279dbbfbeb2ebd9ec8d4dbd01e22/webpack/resolve.js#L7
This causes the standalone build to be bundled, which is useful to parse runtime template tags.
But since we are using `vue-loader`, which compiles all template tags in vdom render functions, the runtime build will suffice.... and is 1/3rd smaller!

https://github.com/vuejs/vue/wiki/Vue-2.0-RC-Starter-Resources#standalone-vs-runtime-builds

> When you use vue-loader or vueify to import *.vue files, their <template> parts are automatically compiled into render functions. It is therefore recommended to use the runtime-only build with *.vue files.

> The benefit is that it is roughly 1/3 lighter-weight than the standalone build.

Is there any reason we _do_ include the full standalone build?

**Comment by MatteoGabriele** [comment]
I believe is because previously we were using an .css .html and .js file approach, then we switched to a .vue file because `vue-loader` was handling css-modules out of the box.

We should look into it :)

**Comment by ThaNarie** [comment]
FYI, removed that line in my project, and still works fine :)

**Comment by MatteoGabriele** [comment]
Well I actually have an error removing that line because the main app is still loaded with the template approach, so I should make a little refactor on the way we mount the app.

**Comment by ThaNarie** [comment]
What route do you want to go? Transform the template into a render function?
```
export default {
  // template,
  render (c) {
    return c('router-view')
  },
  store,
  router
}
```
Or convert it into a `.vue` file?

**Comment by MatteoGabriele** [comment]
yeah I would like to make a .vue file with a .js file, like a normal component. sometimes you need to put more than the router-view in that file.

**Comment by ThaNarie** [comment]
Seems like converting it into a `.vue` file works better, allows you to make use of the template for an application shell :)
Using a render function makes it basically useless (except if we want to use jsx there, but probably should not mix the two within 1 project)

**Comment by SoullessWaffle** [comment]
I recently switched to the runtime-only build on my (WIP) chrome extension. It was a [fairly straightforward change](https://github.com/SoullessWaffle/yaglman/commit/6670fe55d40fd8589e03932b28313b42ff8c6949), all it took was moving the root app code to a `.vue` file and importing and mounting it in the bootstrapping file.
Hopefully doing the same with Blue will be similarly straightforward.

**Comment by ThaNarie** [comment]
I did the same in my current project, and works just fine.
Anyone working on a PR already, otherwise I can start one..
